### PR TITLE
Fix watchOS debug log format and delete redundant code in test apps

### PR DESCRIPTION
**Pull request by doudounan**
- Fix watchOS debug log format.
- Delete redundant code in test apps

### CODE REVIEW

File: GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestAppWatchKitExtension/InterfaceController.swift [hunk 1]
```diff
@@ -21,7 +21,8 @@ import Dispatch
 import GoogleDataTransport
 
 class InterfaceController: WKInterfaceController {
-  let transport: GDTCORTransport = GDTCORTransport(mappingID: "1234", transformers: nil, target: GDTCORTarget.test.rawValue)!
+  let transport: GDTCORTransport = GDTCORTransport(mappingID: "1234", transformers: nil,
+                                                   target: GDTCORTarget.test.rawValue)!
```
Comment (mikehaney24) [inline GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestAppWatchKitExtension/InterfaceController.swift:25]: If you choose test, no network request will be sent. Is that intentional?

File: GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -278,13 +304,60 @@
...
+/* Begin PBXShellScriptBuildPhase section */
+		0E8625BE155C7EE0E873F36C /* [CP] Check Pods Manifest.lock */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+				"${PODS_PODFILE_DIR_PATH}/Podfile.lock",
```
Comment (chliangGoogle) [inline GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestApp.xcodeproj/project.pbxproj:316]: these look like not needed to checkin. have you run `pod deintegrate`

File: GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestAppWatchKitExtension/InterfaceController.swift (hunk 1 above)
Comment (doudounan) [reply GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestAppWatchKitExtension/InterfaceController.swift:25]: This test app is in GDT not GDTCCT, I think it is intentional not to sent any request.

File: GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestApp.xcodeproj/project.pbxproj (hunk 2 above)
Comment (doudounan) [reply GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestApp.xcodeproj/project.pbxproj]: deintegrate in new commit. Thanks!

**Pull request merged**